packages/ActionBar/tests/Actionbar.cypress.js

```diff
@@ -0,0 +1,96 @@
+describe("ActionBar", () => {
+  beforeEach(() => {
+    cy.visitStorybook("action-bar--showcase");
+  });
+
+  it("Should render", () => {
+    cy.get("span")
```

Comment:
I think using `data-pka-anchor` is better here, you can use `cy.getByTestId('raw-button')`